pkg/activator/handler/handler.go

```diff
@@ -49,7 +49,7 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ar := a.Activator.ActiveEndpoint(namespace, name)
 	if ar.Error != nil {
 		msg := fmt.Sprintf("Error getting active endpoint: %v", ar.Error)
-		a.Logger.Error(msg)
+		a.Logger.Errorf(msg)
```

Comment:
Why change this to `Errorf`? If anything I think this should change to

```golang
a.Logger.Errorw("Error getting active endpoint", zap.Error(ar.Error))
```

Which would be in-line with the rest.

Reply:
I don't remember why I touched this file. Could be a mistake. Reverted.